README.md

```diff
@@ -1,42 +1,50 @@
-
-#### Open features
-- [ ] Baking different masks to images (Bevel, Pointiness, Curvature, ...)
-- [ ] Baking masks / mask stacks to a single image mask
-- [ ] Other layer types (for example a paint only layer or a material layer)
-- [ ] PBR Texture assets
-- [ ] Brush assets for texture painting
-- [ ] Different channel presets for NPR, PBR, ...
-
-## Changelog
-#### v2.0.0
-- Initial release
+# Layer Painter
+
+Layer Painter is a addon for blender, made to bring a workflow similar to a tool like substance painter or armor paint directly inside blender.
+The goal here is to provide a workflow which makes the tools mentioned above unnecessary for a lot of projects, keeping the work fully inside of blender.
+
+**[Discord Server](https://discord.com/invite/s9dawzV5JU)**
+
+The discord server is used as the main place to share information and work regarding the addon. You can join it to ask questions and get more information about contributing.
+
+## Overview
+- [Addon](#addon)
+- [Additional Assets](#additional-assets)
+- [Tutorials](#tutorials)
+- [Contributors](#contributors)
+- [Changelog](#changelog)
+
+## Addon
+WIP (This section will contain a written description on how to use the addon)
+
+## Additional Assets
+- [Texture Based Grunge Maps by BongoCat - Already Included](https://drive.google.com/file/d/11sJSfT7jJohLHouMSQB5oc4GzUHtOifO/view)
+
+## Tutorials
+[Layer Painter 2, Complete Tutorial and free scene by blanchsb](https://www.youtube.com/watch?v=YDt9LFbjJU4)
+
+[Layer Painter 2, Materials and Asset Browser Tutorial by blanchsb](https://www.youtube.com/watch?v=glMGVkGhtAg)
+
+## Contributors
+This addon is maintained by contributors. This most importantly includes contributions for code but also for assets like masks and filters and tutorials. If you're interested in contributing any of these, just open a pull request for adding them yourself or join the discord server and ask for more information.
+
+### Code Contributions
+You can fix bugs, add your own feature ideas or other people ideas. If it's something bigger ask on the discord server before you start working to know if someone else already started on it or if you should make changes to your initial idea. When you're done, submit a pull request and your code can be added.
+
+#### Open features
+- [ ] Baking different masks to images (Bevel, Pointiness, Curvature, ...)
+- [ ] Baking masks / mask stacks to a single image mask
+- [ ] Other layer types (for example a paint only layer or a material layer)
+- [ ] PBR Texture assets
+- [ ] Brush assets for texture painting
+- [ ] Different channel presets for NPR, PBR, ...
+
+## Changelog
+#### v2.0.0
+- Initial release
+
+#### v2.0.2

```

Comment:
This should still be 2.0.1 as there is no release for that version yet. Please change here and in package json